Source/Scene/Cesium3DTileset.js

```diff
@@ -892,13 +879,8 @@ define([
 
     function raiseLoadProgressEvents(tileset, frameState) {
         var eventsToRaise = tileset._loadProgressEventsToRaise;
-        var length = eventsToRaise.length;
-        for (var i = 0; i < length; ++i) {
-            var numberOfPendingRequests = eventsToRaise[i].numberOfPendingRequests;
-            var numberProcessing = eventsToRaise[i].numberProcessing;
-
-            frameState.afterRender.push(evenMoreComplicated(tileset, numberOfPendingRequests, numberProcessing));
-        }
+        frameState.afterRender.push(evenMoreComplicated(tileset, tileset._statistics.numberOfPendingRequests,
```

Comment:
Alright so when I don't use `evenMoreComplicated` I'm not passing in a function anymore so I get an error in `Scene.js` because of this function:

<img width="498" alt="screen shot 2016-02-05 at 4 30 25 pm" src="https://cloud.githubusercontent.com/assets/11602092/12859692/f9a431ca-cc25-11e5-9e7e-a4fa5413e62a.png">

If I'm not mistaken since we removed the ability for each tile to add an event and we are only sending __one__ event per frame this function can be refactored and or removed?